Block/Adminhtml/Reindex/Edit/Form.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Module Algolia Algoliasearch
+ */
```

Comment:
Can you remove this comment? It's not needed as it's directly in AlgoliaSearch module directory